e2e/cloud_manager/helper_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e cloudmanager
+
+package cloud_manager_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/mongodb/mongocli/internal/convert"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+const (
+	entity        = "cloud-manager"
+	serversEntity = "servers"
+)
+
+func automatedServer(cliPath string) (string, error) {
```

Comment:
[np] I didn't get much from this name -- automated because it uses the cli? (I cant think of much better though -- `serverHostname`? )

Reply:
fixed and added a doc